repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -28,15 +29,16 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
-
 /**
  * This class is the backbone for the realtime scanner.
  */
 public class RTScanner {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(RTScanner.class);
     private static final int DURATION_IN_TEMP_BLACKLIST = 600; // in seconds
-
+    
+    // by default, the RTScanner does nothing
```

Comment:
this can also go to RepairnatorConfig

Reply:
This can't be removed, since we are using it in the class itself for checking which mode we are running :) .

Reply:
Which the problem that has to be fixed :-). 

You can replace by `if (Repairnator.getInstance().getPipeline().equalds(...)` or something like that.

Reply:
Ah I see what you meant now. I was blind by the lines inside the config file and I completely missed out the singleton implementation methods :) . I'll fix it in a moment.